DataStructures/SinglyLinkedList.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ *  Singly Linked List
+ */
+
+class SinglyLinkedList
+{
+    public ?SinglyLinkedList $next = null;
+
+    public function __construct(public $data)
+    {
+    }
```

Comment:
You need to add a property for $data, and set the property's value in the constructor

Reply:
Thanks for the feedback, Brandon. From your comment, I understand you'd like me to move the definition of $data to the class properties and add setter in the constructor. Adding 'public' in the constructor's property does both of those things but I can see how it might be confusing for some people, especially if they're new to coding.